test/hexer_package_SUITE.erl

```diff
@@ -64,7 +64,8 @@ publish(_Config) ->
   ct:comment("Create dummy files"),
   ok = file:make_dir("src"),
   AppSrcBin = <<"{application, hexer, [{vsn, \"0.0.1\"}]}.">>,
-  ok = file:write_file("src/hexer.app.src", AppSrcBin),
+  HexerAppSrcFileName = "src/hexer.app.src",
```

Comment:
can you name the file `my_app.app.src` instead, to avoid issues if someone runs this test in a shell?